tests/data/mock.4dedup.pairsam

```diff
@@ -4,13 +4,14 @@
 #genome_assembly: unknown
 #chromosomes: chr1 chr2
 #columns: readID chrom1 pos1 chrom2 pos2 strand1 strand2 pair_type sam1 sam2
-.	!	0	chr1	25	-	+	NU	.	.
-.	!	0	chr1	25	-	+	NU	.	.
-.	chr1	1	chr1	20	+	+	UU	.	.
-.	chr1	1	chr1	20	+	+	UU	.	.
-.	chr1	1	chr1	25	+	+	UU	.	.
-.	chr1	1	chr1	27	+	+	UU	.	.
-.	chr1	1	chr1	28	+	-	UU	.	.
-.	chr1	1	chr1	28	+	+	UU	.	.
-.	chr1	1	chr1	50	+	+	UU	.	.
-.	chr1	1	chr2	25	+	+	UU	.	.
+1	!	0	chr1	25	-	+	NU	.	.
```

Comment:
Can we make readID names in the test consistent with other tests?

Reply:
Done!